jpo-ode-core/src/main/java/us/dot/its/jpo/ode/wrapper/MessageProducer.java

```diff
@@ -88,6 +89,12 @@ public MessageProducer(
         props.put("key.serializer", SERIALIZATION_STRING_SERIALIZER);
         props.put("value.serializer", valueSerializerFQN);
 
+        String lingerMsEnv = System.getenv("KAFKA_LINGER_MS");
+        if(lingerMsEnv != null){

```

Comment:
suggestion (non-blocking): to prevent a `NumberFormatException `occurring on startup and crashing the service when `KAFKA_LINGER_MS` is an empty string, we may consider adding to the if check like so. To test for this error locally, use `KAFKA_LINGER_MS=` in your .env file

```suggestion
        if (lingerMsEnv != null && !lingerMsEnv.isEmpty()) {

```

Reply:
I like it.